docs/product-user-guide/src/main/asciidoc/decision-tables-attributes-ref.adoc

```diff
@@ -0,0 +1,67 @@
+[id='decision-tables-attributes-ref']
+= Additional rule attributes for RuleSet or RuleTable definitions
+
+The `RuleSet` and `RuleTable` areas also support labels and values for other rule attributes, such as `PRIORITY` or `NO-LOOP`. Rule attributes specified in a `RuleSet` area will affect all rule assets in the same package (not only in the spreadsheet). Rule attributes specified in a `RuleTable` area will affect only the rules in that rule table. You can use each rule attribute only once in a `RuleSet` area and once in a `RuleTable` area. If the same attribute is used in both `RuleSet` and `RuleTable` areas within the spreadsheet, then `RuleTable` takes priority and the attribute in the `RuleSet` area is overridden.
+
+*<@Michael (6 of 6): A recurring ambiguity all over this doc (you'll see it). In a decision table, you have one RuleSet and any number of RuleTables. What's in the RuleSet applies to all RuleTables within that spreadsheet AND to all other RuleTables in all other uploaded spreadsheets in the entire package? We say multiple times that the RuleSet in the the decision table defines rule attributes for all RuleTables in the whole _package_, not just for all RuleTables in the spreadsheet. So, does that mean you can basically only have one uploaded spreadsheet per package, with all RuleTables in it, since there can only be one RuleSet? Or are we saying you can have as many uploaded decision table spreadsheets as you like in a package, but each one must have the same RuleSet name, definitions, etc.? If that's the case, then we should probably be telling users to just put all RuleTables for the whole package in the same single spreadsheet, with it's single corresponding RuleSet there identified. Right?>*
```

Comment:
@sterobin If the user uploads multiple spreadsheets into the same package and more than one has a RuleTable that uses the same attributes listed in "Table 2.3" then there is definitely room for error. These attributes, when added to the DRL, control Drools rule engine behaviour for the whole package. If XLS1 defines, for example, ENABLED as false and XLS2 defines it as DISABLED then which takes precedence will rely on the order in which the different XLSs are added to the rule engine. So a warning in the documentation should be given about using the attributes in "Table 2.3" in different RuleTables in different XLSs in the same package. If the User uses the same attributes in different XLSs it would be advisable to have them in different packages. The regular RuleTable attributes ("Table 2.1") only apply to the XLS being processed and not the whole package. To be honest I've never seen Table 2.1's attributes used in the RuleTable and had to run some tests here to verify they actually worked! (they do).